### PR TITLE
Adding sass indented syntax support.

### PR DESCRIPTION
**Pull request by arshdkhn1**
I hope these are the only files that I have to change in order to support indented sass syntax. Please point out if I am missing anything.

### CODE REVIEW

File: src/components/HomePage.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import {Link} from 'react-router';
+import '../styles/homepage.sass';
```
Comment (arshdkhn1) [inline src/components/HomePage.js]: A demo file to show sass indented syntax support.

**Comment by coryhouse** [comment]
Great! Can you please remove the actual Sass example though? I figure merely supporting it via the loader is sufficient.

**Comment by arshdkhn1** [comment]
I am not sure what this code does - 
``` js
sassLoader: {
          includePaths: [path.resolve(__dirname, 'src', 'scss')]
        },
```
I don't see any such folder. What does this code do?

**Comment by arshdkhn1** [comment]
@coryhouse removed the sass example.

**Comment by coryhouse** [comment]
Squashed and merged. Thanks!

**Pull request merged**